### PR TITLE
chore(CHANGELOG): update to v9.4.0

### PR DESCRIPTION
**Pull request by nikku**
https://github.com/bpmn-io/bpmn-js/compare/v9.3.2...clg

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,12 +17,17 @@ ___Note:__ Yet to be released changes appear here._
 * `FIX`: prevent bogus meta-data to be attached on paste ([#1707](https://github.com/bpmn-io/bpmn-js/pull/1707))
 * `FIX`: only claim existing IDs ([#1707](https://github.com/bpmn-io/bpmn-js/pull/1707))
 * `FIX`: prevent double paste on label creation ([#1707](https://github.com/bpmn-io/bpmn-js/pull/1707))
+* `FIX`: move labels when collapsing sub-process ([#1695](https://github.com/bpmn-io/bpmn-js/issues/1695))
+* `FIX`: move labels when collapsing sub-process ([#1695](https://github.com/bpmn-io/bpmn-js/issues/1695))
```
[the remark this answers is not in the document]
Comment (barmac) [reply CHANGELOG.md:21]:
```suggestion
```

**Comment by barmac** [comment]
We want to include a pending bpmn-moddle release in this: https://github.com/bpmn-io/bpmn-moddle/pull/95

**Comment by nikku** [comment]
@barmac We can also ship the `bpmn-moddle` changes with a follow-up patch release.

**Comment by nikku** [comment]
Release often! :runner:

File: CHANGELOG.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (nikku) [reply CHANGELOG.md]: Fixed!

**Pull request merged**